sdk/appcenter-analytics/src/main/java/com/microsoft/appcenter/analytics/Analytics.java

```diff
@@ -59,7 +61,8 @@
     /**
      * Shared instance.
      */
-    private static Analytics sInstance = null;
+    @SuppressLint("StaticFieldLeak")
+    private static Analytics sInstance;
```

Comment:
Just curious, what's the reason for this change?

Reply:
It was a warning we always ignore, also no need to init with null as its default value.

Reply:
We don't initialize anything to null since it is by default. And it added `SuppressLint` to ignore lint complains.